src/AdoPat/PatManager.cs

```diff
@@ -19,19 +20,24 @@ public class PatManager
         private const int ValidToExtensionDays = 7;
         private const int NearingExpirationDays = 2;
 
+        // Note: Logging in this class intentionally avoids referencing any PAT details lest those leak into log files.

```

Comment:
@reillysiemens I did not get this comment. Can you elaborate this a little?

Reply:
Sure. Lemme take a stab a rephrasing this and if it's suitable to you I'll update the comment.
```csharp
// Note: Do NOT use this logger instance to log any fields on a PAT, ESPECIALLY the token.
```

Reply:
Yes, it's simpler english I think :)

Reply:
Basically you are saying do not log PAT values?

Reply:
Yes, exactly. Anything logged could end up in a log file. Best to just never log any part of a PAT.